equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/model/report/Crm14CaseSummaryReportModel.java

```diff
@@ -112,6 +113,12 @@ public String exportToCSV() {
         );
     }
 
+    private String sanitiseField(String text) {
```

Comment:
small suggestions, it can be one liner code as below
`return text == null ? null : text.replace(",", ";");`

Reply:
I am not a big fan of 1 liners or terciary operators when the 1st condition is null, false or negative. it basically hides the logic in the end of a long sentence. 
But given the simplicity, I changed it.